src/main/java/bio/terra/service/dataset/flight/ingest/DatasetIngestFlight.java

```diff
@@ -156,6 +157,13 @@ public DatasetIngestFlight(FlightMap inputParameters, Object applicationContext)
 
     addStep(new IngestSetupStep(datasetService, configService, cloudPlatform));
 
+    CloudFileReader cloudFileReader = (cloudPlatform.isGcp()) ? gcsPdao : azureBlobStorePdao;
```

Comment:
Previously we only ran this step for ingests in ARRAY or JSON format to GCP datasets

But we also need to be validating that the user can access the control file when ingesting in CSV format.

It's not strictly necessary that this step run for Azure datasets, but it's safer to run it unconditionally and delegate no-op status to the step where needed.